__test__/api/comments/updateSingleCommentFlags.test.js

```diff
@@ -0,0 +1,132 @@
+const app = require("../../../server");
+const CommentModel = require("../../../models/comments");
+const commentHandler = require("../../../utils/commentHandler");
+const supertest = require("supertest");
+const request = supertest(app);
+
+describe("PATCH /comments/:commentId/flag", () => {
+  let sampleComment, sampleComment2;
+
+  beforeEach(async () => {
+    const comment = new CommentModel({
+      refId: "94736880",
+      applicationId: global.application._id,
+      ownerId: "myuseremail@email.com",
+      content: "this is a very bad comment",
+      origin: "myuseremail@email.com",
+      upVotes: ["sally@gmail.com", "shully@gmail.com", "kelly@gmail.com"],
+      downVotes: ["rally@gmail.com", "molly@gmail.com"],
+    });
+
+    const comment2 = new CommentModel({
+      refId: "576897564",
+      applicationId: global.application._id,
+      ownerId: "anotheruseremail@email.com",
+      content: "this is another comment ",
+      origin: "myuseremail@email.com",
+      upVotes: ["pally@gmail.com"],
+      downVotes: ["tally@gmail.com"],
+    });
+
+    const commentOne = await comment.save();
+    const commentTwo = await comment2.save();
+
+    sampleComment = commentHandler(commentOne);
+    sampleComment2 = commentHandler(commentTwo);
+  });
+
+  afterEach(async () => {
+    await CommentModel.findByIdAndDelete(sampleComment.commentId);
+    await CommentModel.findByIdAndDelete(sampleComment2.commentId);
+
+    sampleComment = null;
+    sampleComment2 = null;
+  });
+
+  test("Flag a single comment. Should return 200", async () => {
+    //check that the db has not been updated
+    await CommentModel.findById(sampleComment.commentId).then((comment) => {
+      expect(comment.flags.length).toEqual(sampleComment.numOfFlags);
+      expect(comment.ownerId).toEqual(sampleComment.ownerId);
+    });
+
+    const res = await request
+      .patch(`/v1/comments/${sampleComment.commentId}/flag`)
+      .set("Authorization", `bearer ${global.appToken}`)
+      .send({
+        ownerId: sampleComment2.ownerId,
+      });
+
+    expect(res.status).toBe(200);
+    expect(res.body.status).toEqual("success");
+
```

Comment:
Don't forget to also test for the data property. It's also suppose to be expected. See the [swagger documentation for endpoint](https://comment.microapi.dev/#/comments/patch_comments__commentId__flag).